### ISSUE TITLE
Websocket for tasks

### ISSUE THREAD

**Issue by Kalinow** [viadee/sonarQuest#266]
Hey,
i think we need a further websocket for Quests/Tasks.
(Or we extend the /chat Websocket but i think this is not a realy good solution :) )

When a user takes part in a quest/task then the view should be updated. So that all players sees the correct participation state.

**Comment by surfermicha** [comment]
I agree with you, that a websocket for another purpose should separated from the existing one.

This might be a good challenge for the next coding night, beacuse I guess that we need "live updates" an much more cases so that we should work on it with a coneceptional view. What do you think?

**Comment by Christoph-Meyer** [comment]
I dig the idea, too! There are quite a few places in the UI where this is necessary.
